src/redisearch_rs/query_error/src/lib.rs

```diff
@@ -255,6 +257,10 @@ impl QueryWarningCode {
             Self::OutOfMemoryCoord => {
                 c"One or more shards failed to execute the query due to insufficient memory"
             }
+            Self::UnavailableSlots => c"Query requires unavailable slots",
+            Self::AsmInaccurateResults => {
+                c"Query execution exceeded maximum delay for RediSearch to delay key trimming. Results may be incomplete or have duplications due to Atomic Slot Migration."
```

Comment:
Where are we using these?

Reply:
These are internals of redisearch_rs, to have complete headers., and to get stats and error conversion from string to error.